extensions/nginx/index.js

```diff
@@ -213,77 +196,81 @@ class NginxExtension extends Extension {
                     port: instance.config.get('server.port')
                 });
 
-                return this.template(instance, generatedSslConfig, 'ssl config', confFile, `${nginxConfigPath}/sites-available`).then(
-                    () => this.ui.sudo(`ln -sf ${nginxConfigPath}/sites-available/${confFile} ${nginxConfigPath}/sites-enabled/${confFile}`)
-                ).catch(error => Promise.reject(new ProcessError(error)));
-            }
+                await this.template(instance, generatedSslConfig, 'ssl config', confFile, `${nginxConfigPath}/sites-available`);
+                await this.ui.sudo(`ln -sf ${nginxConfigPath}/sites-available/${confFile} ${nginxConfigPath}/sites-enabled/${confFile}`);
+            })
         }, {
             title: 'Restarting Nginx',
             task: () => this.restartNginx()
         }], false);
     }
 
-    uninstall({config}) {
+    async uninstall({config}) {
         const instanceUrl = config.get('url');
 
         if (!instanceUrl) {
-            return Promise.resolve();
+            return;
         }
 
         const parsedUrl = url.parse(instanceUrl);
         const confFile = `${parsedUrl.hostname}.conf`;
         const sslConfFile = `${parsedUrl.hostname}-ssl.conf`;
 
-        const promises = [];
+        let restart = false;
```

Comment:
Looks like these commands went from parallel to serial - was that expected?

Reply:
yeah - I didn't think the benefit of running them in parallel outweighed code cleanliness. if these were super slow commands then it might make sense to run them in parallel but since these are just file deletions it's fine to run them in series